### PR TITLE
recipe for pycld2

### PR DESCRIPTION
**Pull request by d-cunningham**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by d-cunningham** [comment]
@JennaLipscomb maintainer?

**Comment by mxr-conda** [comment]
Agree to maintain

**Comment by d-cunningham** [comment]
@conda-forge/staged-recipes, recipe ready for review

**Comment by d-cunningham** [comment]
bump

File: recipes/pycld2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycld2-{{ version }}.tar.gz
+  sha256: a42f6e974df8fdd70685c2baa8a9f523069a260e1140ce604fb9f1fb6c3064df
+  patches:
+    - relative_sources.patch
+
+build:
+  number: 0
+  skip: true  # [win64]
```
Comment (goanpeca) [inline recipes/pycld2/meta.yaml:16]: @d-cunningham should this just be `[win]`?

**Comment by goanpeca** [comment]
@d-cunningham very minor nitpick, otherwise looks good, thanks!

**Comment by goanpeca** [comment]
Thanks for working on the recipe 🚀

**Pull request merged**